app/src/test/java/com/appunite/loudius/ui/reviewers/ReviewersViewModelTest.kt

```diff
@@ -91,104 +98,160 @@ class ReviewersViewModelTest {
         fun `GIVEN correct initial values WHEN init THEN pull request number is correct`() {
             viewModel = createViewModel()
 
-            assertEquals("correctPullRequestNumber", viewModel.state.pullRequestNumber)
+            expectThat(viewModel.state)
+                .get(ReviewersState::pullRequestNumber)
+                .isEqualTo("correctPullRequestNumber")
         }
 
         @Test
         fun `GIVEN correct initial values WHEN init starts THEN state is loading`() = runTest {
-            coEvery { repository.getReviews(any(), any(), any()) } coAnswers { neverCompletingSuspension() }
+            coEvery {
+                repository.getReviews(
+                    any(),
+                    any(),
+                    any(),
+                )
+            } coAnswers { neverCompletingSuspension() }
```

Comment:
**Question:** Was it auto-formatting or do you prefer such a format? Cause it's a lot of additional lines.

Reply:
I think auto formatted :) 
Actually I don't care ;)